pkg/users/account-create-dialog.js

```diff
@@ -179,21 +195,47 @@
         locked: false,
         confirm_weak: false,
         change_passw_force: false,
+        base_home_dir: "",
+        home_dir: "",
+        shell: "",
     };
     let errors = { };
 
     let old_password = null;
     let user_name_dirty = false;
 
+    function get_defaults() {
+        return cockpit.spawn(["/usr/sbin/useradd", "-D"], { superuser: "require" })
+                .catch(() => "")
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230124-140007-d647d806-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)